charts/operator-wandb/charts/weave-trace/values.yaml

```diff
@@ -0,0 +1,60 @@
+nameOverride: ""
+fullnameOverride: ""
+
+# These need to be set as well if app name overrides are set
+app:
+  serviceName: "app"
+  clusterDomain: "cluster.local"
+  nameOverride: ""
+  fullnameOverride: ""
```

Comment:
could you explain what this does and why we need it?

Reply:
We need to resolve the cluster-local hostname for app.  We do something similar in `weave` https://github.com/wandb/helm-charts/blob/6a8df136066d848568b5e3bc7e73ffcc9b01bf21/charts/operator-wandb/charts/weave/values.yaml#L4

Reply:
re: why, `weave-trace` uses gorilla for auth.  we want to avoid hitting external dns and the additional hop routing through the ALB.

Reply:
If we do `http://{{ include "weaveTrace.appFullname" . }}:8080` doesn't that resolve needing this section?

Reply:
yes, confirmed this works. thanks!

Reply:
wait, `weaveTrace.appFullname` itself depends on `app.*`, to get rid of those `app` fields we'd have to do promote the app chart's name-related values to globals, right?

Reply:
We do it like this: https://github.com/wandb/helm-charts/blob/6a8df136066d848568b5e3bc7e73ffcc9b01bf21/charts/operator-wandb/charts/console/templates/deployment.yaml#L57

While not ideal, it simplifies

Reply:
we hardcode the service name

Reply:
now gone